### PR TITLE
Adding custom error collection info to the ET FE standalone docs

### PR DESCRIPTION
**Pull request by meghanlo**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by estherk15** [comment]
Docs editorial review: https://datadoghq.atlassian.net/browse/DOCS-8506

File: content/en/real_user_monitoring/error_tracking/browser.md [hunk 1]
```diff
@@ -45,6 +45,103 @@ Source maps are limited to **500** MB each.
 Source maps are limited to **500** MB each.
 {{< /site-region >}}
 
+## Collect errors manually
+
+For more information about collecting browser errors, check our documentation [here][2]
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/browser.md:50]:
```suggestion
You can monitor handled exceptions, handled promise rejections, and other errors that the Browser SDK does not automatically track. Learn more about [Collecting Browser Errors][2].
```

File: content/en/real_user_monitoring/error_tracking/browser.md [hunk 2]
```diff
@@ -45,6 +45,103 @@ Source maps are limited to **500** MB each.
...
+
+
+Monitor handled exceptions, handled promise rejections, and other errors not tracked automatically by the Browser SDK with the `addError()` API:
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/browser.md:53]:
```suggestion
To manually collect errors, use the `addError()` API:
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Added an intro to the feature, and I am wondering what the purpose of `content/en/error_tracking/standalone_frontend/collecting_browser_errors.md` is, but otherwise looks good.

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**